internal/cli/cmds.go

```diff
@@ -267,6 +267,11 @@ func runAdd(
 	defer span.Finish()
 	b := backends.GetBackend(ctx, language)
 
+	if !b.QuirksCanAddRemove() {
+		fmt.Println("This backend does not suppport dependency additions")
```

Comment:
```suggestion
		fmt.Println("This backend does not suppport dependency addition")
```
nit to keep consistent with the other error message